web/src/components/home/subcomponents/upcoming_events/upcoming_events.js

```diff
@@ -1,5 +1,92 @@
+import React from "react";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Container from "@mui/material/Container";
+import DyosRenderer from "./event_renderers/dyos_renderer";
+import KeyboardArrowLeft from "@mui/icons-material/KeyboardArrowLeft";
+import KeyboardArrowRight from "@mui/icons-material/KeyboardArrowRight";
+import MobileStepper from "@mui/material/MobileStepper";
+import Typography from "@mui/material/Typography";
+import getNextEvents from "../../../../data/events";
+import FeatureFlags from "../../../../infra/feature_flags";
+import upcomingEventsStyles from "./upcoming_events.styles";
+import messages from "./messages";
+
+function TwoEventRenderer(props) {
+  let events = props.events;
+  return (
+    <Box sx={upcomingEventsStyles.twoEventRenderer}>
+      <DyosRenderer event={events[0]} />
+      <DyosRenderer event={events[1]} />
+    </Box>
+  );
+}
+
+function CarouselEventRenderer(props) {
+  let events = props.events;
+  const [currentEventIndex, setCurrentEventIndex] = React.useState(0);
```

Comment:
you can simplify this by updating the import

`import React as React from "react";`

then this just becomes 

```suggestion
  const [currentEventIndex, setCurrentEventIndex] = useState(0);
```

Reply:
Updated to `import { useState } from "react";`